react/features/prejoin/middleware.js

```diff
@@ -70,26 +76,16 @@ MiddlewareRegistry.register(store => next => async action => {
         store.dispatch(setPrejoinVideoMuted(Boolean(action.muted)));
         break;
     }
+
+    case SET_PREJOIN_NAME: {
```

Comment:
Why are we disspatching this action now? Maybe we can just dispatch updateSettings?

Reply:
I have removed the redundant action.